GOVERNANCE.md

```diff
@@ -1,29 +1,32 @@
 # Main Governance Document
 
+## The Project
 
-The Project
-===========
-
-The PyMC3 Project (The Project) is an open source software project
+The PyMC Project (The Project) is an open source software project
 affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to
 develop open source software and deploy open and public websites and services
 for reproducible, exploratory and interactive computing. The Software developed
-by The Project is released under the Apache 2 open source license,
+by The Project is released under OSI approved open source licenses,
 developed openly and hosted in public GitHub repositories under the
-[GitHub organization](https://github.com/pymc-devs/pymc3). Examples of
-Project Software include the PyMC3 code and the Documentation, etc. The Services run by the
-Project consist of public websites and web-services that are hosted
-at [http://pymc-devs.github.io/pymc3/](http://pymc-devs.github.io/pymc3/)
+[GitHub organization](https://github.com/pymc-devs). Examples of
+Project Software include the PyMC3 library and its documentation, etc.
+The Services run by the Project consist of public websites and web-services
+that are hosted at [http://docs.pymc.io](http://docs.pymc.io)
+
 The Project is developed by a team of distributed developers, called
 Contributors. Contributors are individuals who have contributed code,
-documentation, designs or other work to one or more Project repositories.
+documentation, designs or other work to one or more Project repositories,
+or who have done significant work to empower the Community,
+moderating on [Discourse](http://docs.pymc.io),
```

Comment:
This link does not point to the discourse.

Also what's our correct docs link these days?
docs.pymc.io and pymc3.readthedocs.io don't show the same things.
(We should set up forwarding I guess?)

Reply:
> This link does not point to the discourse.

Thanks, will fix that

> Also what's our correct docs link these days?
docs.pymc.io and pymc3.readthedocs.io don't show the same things.
(We should set up forwarding I guess?)

Docs link is and should be docs.pymc.io. We'll make rtd serve the docs on that same url with the v4 release, and by "we" I mean Sayam who is the rtd wizard